pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -434,4 +434,10 @@ type DisruptionManagementSpec struct {
 	// it only works if managePodBudgetss is true.
 	// the default is 30 minutes
 	OSDMaintenenceTimeout time.Duration `json:"osdMaintenanceTimeout,omitempty"`
+
+	// This enables management of machinedisruptionbudgets
+	ManageMachineDisruptionBudgets bool `json:"manageMachineDisruptionBudgets,omitempty"`
+
+	// Namespace to look for MDBs by the machineDisruptionBudgetController
+	MachineDisruptionBudgetNamespace string `json:"machineDisruptionBudgetNamespace,omitempty"`
```

Comment:
Do we need a RoleBinding in the RBAC to this namespace as well? Otherwise, how does the operator have access to this namespace?

Reply:
yes we need. I have changed the existing roles to cluster roles.

Reply:
I think we need both a cluster role and a normal role. It's not desirable to change the existing permissions to cluster scoped when they're namespace scoped.

Reply:
yeah did as Travis said